src/routes/+layout.svelte

```diff
@@ -72,5 +74,28 @@
 	</svelte:fragment>
 	<!-- Page Route Content -->
 	<slot />
-	<svelte:fragment slot="footer">Page Footer</svelte:fragment>
+	<svelte:fragment slot="pageFooter">
+		<div class="grid grid-cols-1 md:!grid-cols-4 gap-1 lg:ml-10 lg:mr-10">
+			<div>
```

Comment:
why is there a wrapping `<div>` here? If I delete it, the behaviour is the same. Or did I miss something?